apps/user_ldap/lib/User/User.php

```diff
@@ -740,7 +740,7 @@ private function setOwnCloudAvatar() {
 
 
 		//make sure it is a square and not bigger than 128x128
```

Comment:
the comment is now out of date ;)

```suggestion
		//make sure it is a square and not bigger than 512x512
```